droidmediaconstants.cpp

```diff
@@ -97,7 +97,12 @@ void droid_media_colour_format_constants_init(DroidMediaColourFormatConstants *c
   c->OMX_COLOR_Format32bitBGRA8888 = OMX_COLOR_Format32bitBGRA8888;
   c->OMX_COLOR_Format16bitRGB565 = OMX_COLOR_Format16bitRGB565;
   c->OMX_COLOR_Format16bitBGR565 = OMX_COLOR_Format16bitBGR565;
+#if (ANDROID_MAJOR == 4 && ANDROID_MINOR == 4) || ANDROID_MAJOR >= 5
```

Comment:
I don't see the flexible format in Android 4.4 headers https://android.googlesource.com/platform/frameworks/native/+/refs/tags/android-4.4.4_r2.0.1/include/media/openmax/OMX_IVCommon.h

Reply:
Fixed.